### PR TITLE
add m50

### PR DESCRIPTION
**Pull request by ddhogan**

### CODE REVIEW

File: config/default/foia_wizard.settings.yml [hunk 1]
```diff
@@ -156,7 +156,7 @@ messages:
     value: "<p>These records appear to be records of a state or local authority.&nbsp; If you are looking for non-federal records, such as state or local records, we suggest that you contact the proper state or local authority in accordance with the appropriate state or local records access law.</p>\r\n"
     format: rich_text_wizard
   m50:
-    value: "<p>Message 50</p>\r\n"
+    value: "<p>For all other records, we recommend you contact your medical provider directly</p>\r\n"
```
Comment (mrclay) [inline config/default/foia_wizard.settings.yml:159]: Add period.

File: config/default/foia_wizard.settings.yml [hunk 2]
```diff
@@ -156,7 +156,7 @@ messages:
...
     format: rich_text_wizard
   m51:
```
Comment (mrclay) [inline config/default/foia_wizard.settings.yml:161]: We'll use the same message as m50 here.

File: config/default/foia_wizard.settings.yml
Comment (mrclay) [inline config/default/foia_wizard.settings.yml:1]: Let's copy this file to `docroot/modules/custom/foia_wizard/config/install/foia_wizard.settings.yml` as well so they're in sync. Does that seem wise @mattdanger ?

**Review by mrclay — CHANGES_REQUESTED** [review]

File: config/default/foia_wizard.settings.yml
Comment (mattdanger) [reply config/default/foia_wizard.settings.yml:1]: @mrclay yes please!

File: config/default/foia_wizard.settings.yml (hunk 2 above)
Comment (ddhogan) [reply config/default/foia_wizard.settings.yml:161]: I just noticed the pronoun difference on this one (`your` vs. `the`), I'll update the front-end too to use this one when it's for someone else's records

**Pull request merged**